en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,470 @@
+
+            2. In the **Open API Console** pane, copy the API endpoint.
+
+            3. Add `/books` to the end of the API endpoint you copied to complete the resource server URL.
+
+        - `VITE_CHOREO_CLIENT_ID`
+
+            1. Access the Choreo Developer Portal at [https://devportal.choreo.dev/](https://devportal.choreo.dev/).
+
+            2. In the top menu, click **Applications** and then click the **readingListApp** application.
+
+            3. In the left navigation menu, click **Production Keys**.
+
+            4. Copy the value in the **Consumer Key**field and paste it as the value of the `VITE_CHOREO_CLIENT_ID` parameter.
+
+        - `VITE_ORG_HANDLE`
+
+            This is the value displayed next to the **Asgardeo** logo in the Asgardeo Console.
+
+            ![Organization handle](../assets/img/get-started/organization-handle.png){.cInlineImage-threeQuarters}
+
+        - `VITE_STS_TOKEN_ENDPOINT`
+
+            1. In the Choreo Developer Portal, open the **readingListApp** application.
+
+            2. Copy the value in the **Token Endpoint** field and paste it as the value of the `VITE_STS_TOKEN_ENDPOINT` parameter.
+
+Now you have configured your front-end application. Next, you need to configure Asgardeo as the identity provider for your application.
+
+#### Step 2.3.3: Configure Asgardeo as an identity provider for Choreo
+
+Let's deploy a pre-designed front-end application. This application is designed to personalize the readingList based on the user ID that it obtains from its identity provider. To enable the application to obtain the user ID, let's configure Asgardeo as the identity provider by following the steps given below:
+
+1. In the Choreo Console, open the **Home** page.
+
+2. In the left navigation menu, click **Settings**.
+
+3. Click API Management and under **Identity Providers**, click **+ Add Provider**.
+
+4. In the list of identity providers, click **Asgardeo**.
+
+5. In the **Asgardeo** dialog, enter the following information:
+
+    1. In the **Name** and **Description** fields, enter a name and a description for the identity provider.
+
+    2. To get the well-known URL, follow the steps given below:
+    
+        1. Navigate to Asgardeo and open the single-page application you created in [Step 2.4.2: Configure the front-end application](#step-242-configure-asgardeo-as-an-identity-provider).
+
+        2. Click **Info** to view the endpoints of the application.
+
+        3. Copy the endpoint in the **Discovery** field.
+
+        4. Navigate to the Choreo Console and paste the URL you copied in the **Well-Known URL** field in the **Asgardeo** dialog.
+
+    3. Click **Next**, and then click **Add**.
+    
+Now you have successfully configured Asgardeo as the identity provider for the front-end application.
+
+#### Step 2.3.4: Create a user in Asgardeo
+
+To define a user for the readingListApp application, follow the steps given below:
```

Comment:
Who does this? is this done by someone who has admin privileges?

Reply:
Users can self-register of get an admin to onboard them. Updated the introductory paragraph in this section accordingly.

Reply:
The note doesn't add much value to the user. Shall we remove it?

Reply:
Removed it.